hazelcast/src/test/java/com/hazelcast/test/AbstractParameterizedHazelcastClassRunner.java

```diff
@@ -141,8 +140,4 @@ protected Statement classBlock(RunNotifier notifier) {
         }
     }
 
-    @Override
-    protected Annotation[] getRunnerAnnotations() {
```

Comment:
What was an initial idea/purpose to "empty" the annotations in this runner? If we remove it and make the annotations available in the `Description` class wouldn't we break that initial idea.

Reply:
I don't think the original idea was to hide annotations. Or at least it doesn't make sense to me. I would guess rather some copy/paste approach.

Class introduced in:
https://github.com/hazelcast/hazelcast/commit/4333ccc97d36f0d6afcc052167b2bb8e3e7b00fd#diff-e96aa8ebdefa54e0070f67aa3bc02075